index.html

```diff
@@ -282,6 +282,7 @@ <h4>LET'S CONNECT</h4>
       </section>
 
       <section class="contact-me">

```

Comment:
- [ ] As mentioned by the previous reviewer, please make the changes so your design looks like Figma's design. Your design looks like this right now. 

![image](https://user-images.githubusercontent.com/58504655/168115016-5eb1cfc8-6e25-4533-a172-bbac102d2a73.png)